src/Libraries/Microsoft.Extensions.Http.AutoClient/AutoClientUtilities.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Http.AutoClient;
+
+/// <summary>
+/// Utilities for AutoClient feature.
+/// </summary>
+public static class AutoClientUtilities
```

Comment:
Let's mark this with EditorBrowsable.Never since it is not intended for use by humans, and only by the code generator.

Reply:
Done